Extension.php

```diff
@@ -11,10 +12,40 @@ public function initialize()
     {
         $this->addCss('assets/extension.css');
         $this->addJavascript('assets/start.js', true);
```

Comment:
As part of what you're doing, can you move these to a `$this->app->before(array($this, 'before'))`